### PR TITLE
Update plugins.md

### PR DESCRIPTION
**Pull request by srechamp**
Removed "with" from line 12. 👍🏾

### CODE REVIEW

File: pages/plugins.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Add plugins to [command steps](/docs/pipelines/command-step) in your YAML pipeli
 
 Plugins are small self-contained pieces of extra functionality that help you customize Buildkite to your specific workflow. Plugins modify your build [command steps](/docs/pipelines/command-step) at one or more of the ten [job lifecycle hooks](/docs/agent/v3/hooks). Each hook changes a different part of how your jobs set up the environment, check out code, run commands, handle artifacts or clean up the environment.
 
-Reference plugins in your pipeline configuration, and when the step containing the plugin runs, your agent will override the default behavior of with the hooks defined in the plugin [hooks](/docs/agent/v3/hooks).
+Reference plugins in your pipeline configuration, and when the step containing the plugin runs, your agent will override the default behavior of the hooks defined in the plugin [hooks](/docs/agent/v3/hooks).
```
Comment (mbelton-buildkite) [inline pages/plugins.md:12]: @pzeballos, could you confirm whether this should be:

1. ..., your agent will override the default behavior _of_ the hooks defined in the plugin hooks.
2. ..., your agent will override the default behavior _with_ the hooks defined in the plugin hooks.

**Review by mbelton-buildkite — COMMENTED** [review]
@srechamp, thanks for picking up this typo! I'll get someone on the team to check the technical correctness, then I'll run the required checks and merge for you. 😊

File: pages/plugins.md (hunk 1 above)
Comment (mbelton-buildkite) [reply pages/plugins.md:12]: Thanks! @srechamp, are you happy to accept @pzeballos' suggestion? After that, I can run the linting for you and merge. 😊

Comment (srechamp) [reply pages/plugins.md:12]: Np, yep! LGTM 👍🏾

**Review by mbelton-buildkite — APPROVED** [review]

**Review by pzeballos — APPROVED** [review]

**Comment by srechamp** [comment]
Hey @mbelton-buildkite can I delete this branch?

**Comment by mbelton-buildkite** [comment]
> can I delete this branch?

@srechamp, yip! You can delete the branch 😊

**Pull request merged**